contrib/JLightning/src/de/renepickhardt/ln/JLightning.java

```diff
@@ -0,0 +1,36 @@
+package de.renepickhardt.ln;
+
+/**
+ * JLightning a small test / example class to demonstrate how to use JLightningRpc
```

Comment:
Maybe move this into a `README.md` and add the necessary code snippets that you provide here in the `main` method.